README.md

```diff
@@ -0,0 +1,438 @@
+# PoDoFo Readme
+
+## PoDoFo
+
+- [PoDoFo Readme](#podofo-readme)
+  - [PoDoFo](#podofo)
```

Comment:
These two lines seem unnecessary, can we just remove them (and ideally move the other lines to the left)?